### ISSUE TITLE
Bitcoin white paper 

### ISSUE THREAD

**Issue by abbaincase** [calistus-igwilo/nitda-blockchain-scholarship#6]
Hello sir how can I get the Bitcoin white paper

**Comment by oladetohun1** [comment]
check these ones out

**Comment by oladetohun1** [comment]
https://bitcoin.org/bitcoin.pdf
https://www.ussc.gov/sites/default/files/pdf/training/annual-national-training-seminar/2018/Emerging_Tech_Bitcoin_Crypto.pdf

**Comment by oladetohun1** [comment]
Or do you mean do you download the file ,because the whitepaper is already on the repository @abbaincase

**Comment by Mrbee67** [comment]
Sir i create the account then what next

**Comment by maidoya01** [comment]
Yes Sir, I've already downloaded it on GitHub. ( I've created the account
and downloaded it)

**Comment by oladetohun1** [comment]
click this link to downlload the whitepaper: -> It is direct
https://github.com/calistus-igwilo/nitda-blockchain-scholarship/raw/main/bitcoin-white-paper.pdf


You can thank me later

**Comment by 01Bindalhat** [comment]
I really appreciated for being part of this program, Thank you once again sir

**Comment by abdullahidaladi** [comment]
Sir i create the account what to the next step

**Comment by Zayyana1311** [comment]
Masha Allah

**Comment by MDZAIRU** [comment]
I have created the account so what is the next step sir. Thank you

**Comment by Shehugithub** [comment]
Thanks sir God bless you

**Comment by WilliamsAE** [comment]
Download it and rest and read it well well..
Williams Æ from Abuja.

**Comment by Professormalik** [comment]
Olatunde evening

**Comment by 01Bindalhat** [comment]
How I may current my username

**Comment by Mrbee67** [comment]
Sir i wan open centbee account how do i open it
